server/engine/src/main/java/org/eclipse/lsp/cobol/core/engine/symbols/SymbolsRepository.java

```diff
@@ -126,7 +119,14 @@ private static DefinedAndUsedStructure constructElementsExcludingImplicits(Defin
         ctx.getDefinitions().stream().filter(uriNotImplicit()).collect(Collectors.toList());
     List<Location> usages =
         ctx.getUsages().stream().filter(uriNotImplicit()).collect(Collectors.toList());
-    return new SymbolsRepository.Element("", definitions, usages);
+
+    String name = (ctx instanceof CodeBlockUsageNode)
+            ? ctx.getName()
+              + (((CodeBlockUsageNode) ctx).getOfSection() != null
+                      ? " OF " + ((CodeBlockUsageNode) ctx).getOfSection()
+                      : "")
+            : ctx.getName();
```

Comment:
Speaking of breakpoints...
```suggestion
    String name = ctx.getName();
    if (ctx instanceof CodeBlockUsageNode) {
        final CodeBlockUsageNode node = (CodeBlockUsageNode)ctx;
        final String section = node.getOfSection();
        if (section != null)
            name += " OF " + section;
    }
```